server/main.lua

```diff
@@ -4,7 +4,14 @@ TriggerEvent('esx:getSharedObject', function(obj) ESX = obj end)
 
 RegisterNetEvent('blarglebus:finishRoute')
 AddEventHandler('blarglebus:finishRoute', function(amount)
+    local xPlayer = ESX.GetPlayerFromId(source)
+    
+    if xPlayer.job.name == 'busdriver' then
     ESX.GetPlayerFromId(source).addMoney(amount)
+    else
+        print(('blarglebus: %s attempted to exploit the job!'):format(xPlayer.identifier))
+	xPlayer.kick("You are kicked because of cheat!")
+     end
```

Comment:
Could you fix the indentation in this section, please?

Reply:
yes